spec/MediatorManager+services.yaml

```diff
@@ -1,293 +1,296 @@
-          uuid: xx-1-0-0-op-s-bm-010
+          uuid: mm-1-0-0-op-s-bm-010
         - operation-name: /v1/update-operation-client
-          uuid: xx-1-0-0-op-s-bm-011
+          uuid: mm-1-0-0-op-s-bm-011
       individual:
         - operation-name: /v1/bequeath-your-data-and-die
-          uuid: xx-1-0-0-op-s-im-000
+          uuid: mm-1-0-0-op-s-im-000
     service:
       basic:
         - operation-name: /v1/start-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-000
+          uuid: mm-1-0-0-op-s-bs-000
         - operation-name: /v1/inform-about-application
-          uuid: xx-1-0-0-op-s-bs-001
+          uuid: mm-1-0-0-op-s-bs-001
         - operation-name: /v1/inform-about-application-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-002
+          uuid: mm-1-0-0-op-s-bs-002
         - operation-name: /v1/inform-about-release-history
-          uuid: xx-1-0-0-op-s-bs-003
+          uuid: mm-1-0-0-op-s-bs-003
         - operation-name: /v1/inform-about-release-history-in-generic-representation
-          uuid: xx-1-0-0-op-s-bs-004
+          uuid: mm-1-0-0-op-s-bs-004
         - operation-name: /docs
-          uuid: xx-1-0-0-op-s-bs-005
+          uuid: mm-1-0-0-op-s-bs-005
       individual:
-        ## TODO: Add Services specific to this application
+        - operation-name: /v1/create-mediator-kind
+          uuid: mm-1-0-0-op-s-is-000
+        - operation-name: /v1/delete-mediator-kind
+          uuid: mm-1-0-0-op-s-is-001
+        - operation-name: /v1/create-pairing-definition
+          uuid: mm-1-0-0-op-s-is-002
+        - operation-name: /v1/delete-pairing-definition
+          uuid: mm-1-0-0-op-s-is-003
+        - operation-name: /v1/export-pairing-definition-list
+          uuid: mm-1-0-0-op-s-is-004
+        - operation-name: /v1/provide-mediator-for-this-device
+          uuid: mm-1-0-0-op-s-is-005
+        - operation-name: /v1/dismantle-mediator-for-this-device
+          uuid: mm-1-0-0-op-s-is-006
+        - operation-name: /v1/export-mediator-vm-list
+          uuid: mm-1-0-0-op-s-is-007
+
   http-server:
-    own-application-name: ## TODO: Add OwnApplicationName
-    own-release-number: ## TODO: Add OwnReleaseNumber (e.g. 1.0.0)
-    uuid: xx-1-0-0-http-s-000
+    own-application-name: MediatorManager
+    own-release-number: 1.0.0
+    uuid: mm-1-0-0-http-s-000
   tcp-server:
     - description: Without TLS layer
       local-protocol: http
-      own-ip-address: ## TODO: Add OwnIpAddress
-      own-tcp-port: ## TODO: Add OwnTcpPort
-      uuid: xx-1-0-0-tcp-s-000
+      own-ip-address: 1.1.4.6
+      own-tcp-port: 4006
+      uuid: mm-1-0-0-tcp-s-000
     - description: With TLS layer
       local-protocol: https
-      own-ip-address: ## TODO: Add OwnIpAddress
-      own-tcp-port: ## TODO: Add OwnTcpPort
-      uuid: xx-1-0-0-tcp-s-001
+      own-ip-address: 1.2.4.6
+      own-tcp-port: 4206
+      uuid: mm-1-0-0-tcp-s-001
 
 
 clients:
 
```

Comment:
you only need the ElasticSearch client stack if you plan to store information inside an ElasticSearch database; if not, you can delete it

(so first you would need to check if there is any data you want to store persistently, and in case you want to do that you have 2 options: either ElasticSearch or inside a file; if you would go for a file, you can also delete ES-client stack)